app/components/password/reset-password.js

```diff
@@ -11,5 +12,35 @@ export default Component.extend({
    * @property passwordConfirmation
    * @default String
    */
-  passwordConfirmation: ''
+  passwordConfirmation: '',
+  /**
+   * @property error
+   */
+  error: null,
+
+  /**
+   * @property resetPasswordTask
```

Comment:
@begedin Could you review this file?  Was wondering what you thought of it.  Also, how should I handle errors here?  This displays the error as you mentioned in the slack a while back.  But have no way of getting rid of it.